src/components/Button.jsx

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable react/prop-types */
```

Comment:
- [ ] Kindly do not disable linters as they help us write cleaner code.

Reply:
@Ghiftee Thanks for your comment, So if I remove that, can you please tell me how to solve this eslint error
![image](https://user-images.githubusercontent.com/73453971/227729824-b83b324d-d85b-45d7-b124-c610ed95a70c.png)

Reply:
Hi @BahirHakimy , have you tried adding definitions for your prop-types, something like
Display.propTypes = {
  text: PropTypes.string,
  styles: {},
};

Reply:
@Ghiftee Yeah, please check the screenshot below